### DISCUSSION TITLE
I have no stars and am following nobody, yet it's recommending based on "people you follow"

### DISCUSSION THREAD

**Discussion by magcius** [community/community#13579] [Other Feature Feedback, Questions, & Ideas]
![image](https://user-images.githubusercontent.com/40890/160247495-6d65962e-9997-48cf-b513-c6b62ba4f388.png)
![image](https://user-images.githubusercontent.com/40890/160247501-0160d168-1261-4e18-a9af-b1cd27f5383b.png)

This is worrying. Where is it pulling this data from?

**Comment by RGBCube** [comment]
microsoft

**Reply by ZnavoArdai** [reply]
your microsoft account i think

**Comment by jfullstackdev** [comment]
this is almost similar to seeing your own repos as Recommended and recommending other repos although it's already starred, and Staff members are already aware of this

**Comment by loneicewolf** [comment]
was about to ask this question myself, thank gosh someone also has the same question :)
